packages/amplify-category-auth/resources/service-walkthrough-types/cognito/index.ts

```diff
@@ -0,0 +1,122 @@
+export type ServiceQuestionsResult = ServiceQuestionsBaseResult &
```

Comment:
Do we need to export all types in index or just one Auth parent type?

Reply:
we only need override types to be exported to the module . We don't want to expose servicewalkthrough-types